src/Ledger/Certs/Properties.agda

```diff
@@ -312,5 +312,3 @@ module _  {Γ : CertEnv}
                                                    (cong  (_+ getCoin (CertEnv.wdrls Γ))
                                                           (trans (CERT-pov x) (sts-pov sts)))
 
--- TODO: Prove the following property.
--- range vDelegs ⊆ map (credVoter DRep) (dom DReps)
```

Comment:
Why remove this comment? Is this proved somewhere already or is there a ticket for this (so we don't forget)

Reply:
I'll make another property module for this.